### DISCUSSION TITLE
How should I access my camera?

### DISCUSSION THREAD

**Discussion by Dimension4** [bevyengine/bevy#1886] [Q&A] [answered]
I'm new to gamedev and ECS in general so basically I have no idea what I'm doing.

I have a 2D game with an orthographic camera and an UI camera and I want to implement camera controls (drag and scroll), but only for the orthographic camera. How can I make sure to only operate on the ortho camera?

If I use `Query<(&mut Transform, &mut Camera)>` I'll get both cameras. Should I store the ID in a resource, like:

```rust
struct World {
    camera: Option<Entity>
}
```
or should I add a tag type to my camera and use `Query<(&mut Transform, &mut Camera), With<SomeTagType>>`? 

Are there other solutions? What's the way to go here?

**Comment by alice-i-cecile** [comment, marked as answer]
See #1854; this is a known issue that would be a simple fix in Bevy's source :) For now, you need to examine the camera's name field to distinguish them, or like you said, add a tag type to your camera entities.

**Reply by Dimension4** [reply]
That was quick. Thank you!

**Reply by alice-i-cecile** [reply]
We've hooked up Discussions into our Discord channel now! :D

**Reply by FrankenApps** [reply]
I think this is [what I stumbled over](https://stackoverflow.com/questions/67031681/how-to-use-multiple-cameras-in-bevy-0-5-0) too.
